x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/optimizer/LogicalPlanOptimizer.java

```diff
@@ -165,6 +170,7 @@ protected static Batch<LogicalPlan> operators() {
             new CombineBinaryComparisons(),
             new CombineDisjunctions(),
             new SimplifyComparisonsArithmetics(DataType::areCompatible),
+            // new ReplaceStatsAggExpressionWithEval(),
```

Comment:
This is not a leftover. This is a test I did for https://github.com/elastic/elasticsearch/issues/113393. The rules in these two batches need to be cleaned up, unfortunately [not easy to do](https://github.com/elastic/elasticsearch/pull/112392/files#r1738501593).